src/TimeService.Data/WorkTimeMonthLimitRepository.cs

```diff
@@ -28,6 +38,53 @@ public Guid Add(DbWorkTimeMonthLimit workTimeMonthLimit)
             return workTimeMonthLimit.Id;
         }
 
+        public bool Edit(Guid workTimeMonthLimitId, JsonPatchDocument<DbWorkTimeMonthLimit> request)
+        {
+            DbWorkTimeMonthLimit dbWorkTimeMonthLimit = _provider.WorkTimeMonthLimits.FirstOrDefault(l => l.Id == workTimeMonthLimitId)
```

Comment:
better not use simple `l`,  because this can be confused with number 1

Reply:
fixed